test/testkit/mocks/backend/backend.go

```diff
@@ -65,6 +67,12 @@ func WithTLS() Option {
 	}
 }
 
+func ExcludeAPIAccessPort() Option {
```

Comment:
Let's do it unconditionally (always) instead of providing a method to configure it.